src/web/routes/application/flow-control/state-machine/session-accessors.test.js

```diff
@@ -0,0 +1,99 @@
+const test = require('tape')
+const { JOURNEYS_KEY, NEXT_ALLOWED_PATH_KEY, STATE_KEY } = require('./keys')
+const { setNextAllowedPathInSession, setStateInSession, getNextAllowedPathFromSession, getStateFromSession } = require('./session-accessors')
+
+const JOURNEY = { name: 'report-a-change' }
```

Comment:
Could we rename this to REPORT_A_CHANGE_JOURNEY? I think it would make the tests a little bit quicker to read.